_docs/check-balance/overview.md

```diff
@@ -13,7 +13,7 @@ currently in one cold storage address.
 for details). If you've lost access to it, you'll need to recreate a new
 electronic copy by transcribing one of the hardcopies (stored with each private
 key) by hand.
-2. Go to [Blockr](https://www.coinbase.com/), paste your
+2. Go to [Blockstream](https://blockstream.info/), paste your
 <span class="warning">cold storage address</span> into the search bar, and
 press Enter.
 3. You'll be taken to a page that says "Bitcoin Address" at the top, with your
```

Comment:
Blockstream.info does not list anything called "Balance"...the number we're looking for is labeled as "Confirmed Unspent".

Reply:
updated wording